test/webgl/vertex-array.spec.js

```diff
@@ -57,9 +57,7 @@ test('WebGL#VertexAttributes#enable', t => {
     vertexAttributes.disable(i);
   }
 
-  t.equal(vertexAttributes.getParameter(GL.VERTEX_ATTRIB_ARRAY_ENABLED, {location: 0}), true,
-    'vertex attribute 0 should **NOT** be disabled');
-  for (let i = 1; i < MAX_ATTRIBUTES; i++) {
+  for (let i = 0; i < MAX_ATTRIBUTES; i++) {
```

Comment:
See comments above. The received wisdom is that WebGL breaks on some older platforms if we disable attribute 0 so we should avoid it when we can.

Reply:
Generic attribute will not work if the attribute is not disabled. Does this mean that we need to warn user if they attempt to `setGeneric` on location 0?

Reply:
> Does this mean that we need to warn user if they attempt to setGeneric on location 0?

I am not sure. It is possible that the platforms that have an issue with disabling location 0 do not have that issue if a generic attribute has been set on that location. 

Most apps don't use generic attributes so I wouldn't worry about issuing a warning.